static/js/webrtc.js

```diff
@@ -119,35 +154,29 @@ var rtc = (function() {
       $("#options-enablertc").prop("checked", true);
       if (isActive) return;
       self.show();
-      if (isSupported) {
-        padcookie.setPref("rtcEnabled", true);
-        self.getUserMedia();
-      } else {
-        self.showNotSupported();
```

Comment:
Unless `isSupported` is getting set somewhere outside of this module, this function wasn't getting called.